ui/pages/journal.tsx

```
@@ -1,178 +1,97 @@
-                ...journalEntry,
-                createdAt: new Date(journalEntry.createdAt),
-                editedAt: new Date(journalEntry.editedAt),
-              }
-            })
-            // sort habits by createdAt latest first
-            journalEntries.sort((a: any, b: any) => {
-              return b.createdAt.getTime() - a.createdAt.getTime()
-            })
+const parseHabitsToSelectOptions=():SelectOptions[]=>{
+let options:SelectOptions[]=[{value:'all',text:'*All'},{value:'',text:'*No habit'}]
+habits&&habits.forEach((habit:Habit)=>{options.push({value:habit.id,text:habit.title})})
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)